app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java

```diff
@@ -97,7 +97,9 @@ public ConversationItem(Conversation conversation, User user,
     public boolean equals(Object o) {
         if (o instanceof ConversationItem) {
             ConversationItem inItem = (ConversationItem) o;
-            return conversation.equals(inItem.getModel()) && status.equals(inItem.status);
+            return conversation.equals(inItem.getModel()) &&
+                (status == null && inItem.status == null) ||
+                (status != null && status.equals(inItem.status));
```

Comment:
You can use `Objects.equals(status, inItem.status)` instead, it's null safe. I can't add it as a suggestion because it requires the `Objects` import

Reply:
@AlvaroBrey DONE and pushed 👍 feel free to re-review/approve :)